### PR TITLE
Fix secure-console 404

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: kurl_proxy/Makefile [hunk 1]
```diff
@@ -1,5 +1,11 @@
 SHELL := /bin/bash -o pipefail
-export GO111MODULE=on
+export GO111MODULE=off
```
Comment (marccampbell) [inline kurl_proxy/Makefile]: I don't understand this change, is this something from your env?

File: kurl_proxy/cmd/main.go [hunk 2]
```diff
@@ -191,6 +191,15 @@ func getHttpServer(fingerprint string) *http.Server {
...
 		})
 	})
+	r.NoRoute(func(c *gin.Context) {
+		target := url.URL{
+			Scheme:   "https",
+			Host:     c.Request.Host,
+			Path:     c.Request.URL.Path,
+			RawQuery: c.Request.URL.RawQuery,
+		}
+		c.Redirect(http.StatusMovedPermanently, target.String())
```
Comment (marccampbell) [inline kurl_proxy/cmd/main.go]: @divolgin are you ok with this being a 301 vs a 302?

File: kurl_proxy/Makefile (hunk 1 above)
Comment (emosbaugh) [reply kurl_proxy/Makefile]: i was experimenting with something. makes the build a lot faster. i backed it out

File: kurl_proxy/cmd/main.go (hunk 2 above)
Comment (emosbaugh) [reply kurl_proxy/cmd/main.go]: i just went with this here
https://github.com/unrolled/secure/blob/v1/secure.go#L321-L324

Comment (emosbaugh) [reply kurl_proxy/cmd/main.go]: this seems more official though. perhaps ill go with 301
https://support.google.com/webmasters/answer/6073543?hl=en

Comment (emosbaugh) [reply kurl_proxy/cmd/main.go]: oops, that is what i have!

**Pull request merged**